app/javascript/chat/channelDetails.jsx

```diff
@@ -114,21 +114,47 @@ class ChannelDetails extends Component {
     let pendingInvites = [];
     if (channel.channel_mod_ids.includes(window.currentUser.id)) {
       // eslint-disable-next-line
-      searchedUsers = this.state.searchedUsers.map(user => (
-        <div className="channeldetails__searchedusers">
-          <a href={user.path} target="_blank" rel="noopener noreferrer">
-            {user.title}
-          </a>
-          {' '}
+      searchedUsers = this.state.searchedUsers.map(user => {
+        let invite = (
           <button
             type="button"
             onClick={this.triggerInvite}
             data-content={user.id}
           >
             Invite
           </button>
-        </div>
-      ));
+        );
+        if (channel.channel_users.includes(user)) {
+          invite = (
+            <span>
+              is already in
+              {channel.name}
+            </span>
+          );
+        } else if (channel.pending_users_select_fields.includes(user)) {
+          invite = (
+            <span>
+              has already been invited to
```

Comment:
I'd say instead of `has already been invited to`, let's instead not show the user at all here.

So check whether the user exists in the mapping and not show already-invited ones.

Reply:
I've changed it so that it checks if the user has been invited already, and if so, doesn't generate a result row and moves to the next user.